src/runtime-rs/crates/hypervisor/README.md

```diff
@@ -0,0 +1,79 @@
+# Multi-vmm support for runtime-rs
+Some key points for supporting multi-vmm in rust runtime.
+## 1. Hypervisor Config
+The diagram below gives an overvow for hypervisor config
+
+![hypervisor config](../../docs/images/hypervisor-config.svg)
+
+VMM's config info will be loaded when initialize the runtime instance, there are some important functions need to be focused on. 
+### VirtContainer::init()
+[https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/crates/runtimes/virt_container/src/lib.rs#L36](https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/crates/runtimes/virt_container/src/lib.rs#L36)
```

Comment:
URL should not include `main`, please use a tag or commit id instead.

Reply:
I add code blocks to substitute that